aws/devnet/main.tf

```diff
@@ -9,7 +10,23 @@ module "devnet" {
     regions            = var.aws_region
     instance-count     = var.instance_count["bootstrap"]
     docker-org         = "subspace"
-    docker-tag         = "snapshot-2023-aug-15"
+    docker-tag         = "snapshot-2023-sep-25"
+    reserved-only      = false
+    prune              = false
+    genesis-hash       = ""
```

Comment:
if this is empty, then do we need to have this variable ?

Reply:
I use this to set in the `.env` file and populate it when running the node

Reply:
It's populated in the .env file after.